### PR TITLE
{devel}[GCCcore/11.2.0] libtool v2.4.6, Autoconf v2.71, Automake v1.16.4, ...

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

- [x] Depends on #13573

### CODE REVIEW

File: easybuild/easyconfigs/a/Autoconf/Autoconf-2.71-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['431075ad0bf529ef13cb41e9042c542381103e80015686222b8a9d4abef42a1c']
+
+builddependencies = [
+    ('binutils', '2.37'),
+    # non-standard Perl modules are required,
+    # see https://github.com/easybuilders/easybuild-easyconfigs/issues/1822
+    ('Perl.bare', '5.34.0'),
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/a/Autoconf/Autoconf-2.71-GCCcore-11.2.0.eb:27]:
```suggestion
    ('Perl', '5.34.0'),
```

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/0609d08dfa76588e1b971d360234b44f for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
centos8-eb - Linux centos linux 8.3.2011, x86_64, Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (skylake), Python 3.6.8
See https://gist.github.com/01d92e8eca17a7e9f93edba2c959fa28 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
jwvis00.juwels - Linux centos linux 8.3.2011, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 3.6.8
See https://gist.github.com/45258c45b4ff4fce9428a91537963d82 for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
jrlogin11.jureca - Linux centos linux 8.3.2011, x86_64, AMD EPYC 7742 64-Core Processor, Python 3.6.8
See https://gist.github.com/81bf1a8d4fc451b3c2ab89f31a55079b for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
jsfl01.jusuf - Linux centos linux 7.9.2009, x86_64, AMD EPYC 7742 64-Core Processor, Python 3.6.8
See https://gist.github.com/b73e5420dcc58dcf81b20f25e011c8f0 for a full test report.

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @Micket!

**Pull request merged**